### ISSUE TITLE
InstantiationException StringType at startup

### ISSUE THREAD

**Issue by lolodomo** [eclipse/smarthome#1924]
With snapshot 426, at each startup, now I got these 2 warnings:
```
20:30:45.941 [WARN ] [home.core.internal.items.ItemUpdater] - InstantiationException on org.eclipse.smarthome.core.library.types.StringType
20:30:46.325 [WARN ] [home.core.internal.items.ItemUpdater] - InstantiationException on org.eclipse.smarthome.core.library.types.StringType
```
Is it normal ?

**Comment by kaikreuzer** [comment]
No...

**Comment by lolodomo** [comment]
Please reopen this issue, I am still seeing these warnings with offline distribution 430. But it is less frequent, I mean not at each startup.

**Comment by kaikreuzer** [comment]
Could you again provide a log and the exact version of the org.eclipse.smarthome bundles (do "bundle:list -s" on the console)?

**Comment by lolodomo** [comment]
```

openhab> bundle:list -s
START LEVEL 100 , List Threshold: 50
 ID | State    | Lvl | Version               | Symbolic name
-------------------------------------------------------------------------------------------------------
 10 | Active   |  80 | 2.3.0.201506221200    | com.eclipsesource.jaxrs.provider.gson
 11 | Active   |  80 | 5.3.1.201602281253    | com.eclipsesource.jaxrs.publisher
 12 | Active   |  80 | 2.3.1                 | com.google.gson
 13 | Active   |  80 | 18.0.0                | com.google.guava
 14 | Active   |  80 | 3.0.0.v201312141243   | com.google.inject
 15 | Active   |  80 | 1.5.8                 | io.swagger.annotations
 16 | Active   |  80 | 3.19.0.GA             | javassist
 20 | Active   |  80 | 1.1.0.Final           | javax.validation.api
 22 | Active   |  80 | 2.0.1                 | javax.ws.rs-api
 24 | Active   |  80 | 3.2.0.v201101311130   | org.antlr.runtime
 27 | Active   |  80 | 1.6.0                 | org.apache.commons.codec
 28 | Active   |  80 | 3.2.1                 | org.apache.commons.collections
 29 | Active   |  80 | 1.1                   | org.apache.commons.exec
 30 | Active   |  80 | 2.2.0                 | org.apache.commons.io
 31 | Active   |  80 | 2.6                   | org.apache.commons.lang
 32 | Active   |  80 | 3.2.0                 | org.apache.commons.net
 37 | Active   |  80 | 4.2.3                 | org.apache.httpcomponents.httpclient
 38 | Active   |  80 | 4.2.3                 | org.apache.httpcomponents.httpcore
 56 | Active   |  80 | 4.0.4                 | org.apache.karaf.shell.core
 59 | Active   |  80 | 4.0.4                 | org.apache.karaf.wrapper.core
 60 | Active   |  80 | 3.1.0.7               | org.apache.servicemix.bundles.commons-httpclient
 62 | Active   |  80 | 2.11.0.v20150805-0538 | org.eclipse.emf.common
 63 | Active   |  80 | 2.11.1.v20150805-0538 | org.eclipse.emf.ecore
 64 | Active   |  80 | 2.11.1.v20150805-0538 | org.eclipse.emf.ecore.xmi
 65 | Active   |  80 | 3.7.0.v20150402-1709  | org.eclipse.equinox.common
 66 | Active   |  80 | 3.6.0.v20150318-1503  | org.eclipse.equinox.registry
 92 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.config.core
 93 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.config.discovery
 94 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.config.dispatch
 95 | Active   |  75 | 0.9.0.201607271614    | org.eclipse.smarthome.config.xml
 96 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core
 97 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.autoupdate
 98 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.binding.xml
 99 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.id
100 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.persistence
101 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.scheduler
102 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.thing
103 | Active   |  75 | 0.9.0.201607271614    | org.eclipse.smarthome.core.thing.xml
104 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.core.transform
105 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.audio
106 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.console
107 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.console.karaf
108 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.monitor
109 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.net
110 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.rest
111 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.rest.core
112 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.rest.sitemap
113 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.rest.sse
114 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.voice
115 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.core
116 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.item
117 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.item.runtime
118 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.persistence
119 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.persistence.runtime
120 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.rule
121 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.rule.runtime
122 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.script
123 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.script.runtime
124 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.sitemap
125 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.sitemap.runtime
126 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.thing
127 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.model.thing.runtime
128 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.storage.mapdb
129 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.ui
130 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.ui.icon
131 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.ui.iconset.classic
132 | Active   |  80 | 2.9.2                 | org.eclipse.xtend.lib
133 | Active   |  80 | 2.9.2                 | org.eclipse.xtend.lib.macro
134 | Active   |  80 | 2.9.2                 | org.eclipse.xtext
135 | Active   |  80 | 2.9.2                 | org.eclipse.xtext.common.types
136 | Active   |  80 | 2.9.2                 | org.eclipse.xtext.smap
137 | Active   |  80 | 2.9.2                 | org.eclipse.xtext.util
138 | Active   |  80 | 2.9.2.v20160428-1452  | org.eclipse.xtext.xbase
139 | Active   |  80 | 2.9.2                 | org.eclipse.xtext.xbase.lib
155 | Active   |  80 | 2.2.0                 | org.jupnp
156 | Active   |  80 | 1.9.6                 | org.jvnet.mimepull
157 | Active   |  80 | 1.0.9                 | org.mapdb.mapdb
158 | Active   |  80 | 5.0.2                 | org.objectweb.asm
160 | Active   |  90 | 2.0.0.201607210102    | org.openhab.core
161 | Active   |  80 | 2.0.0.201607210102    | org.openhab.core.karaf
167 | Active   |  80 | 1.1.1.201605111122    | com.eclipsesource.jaxrs.provider.swagger
168 | Active   |  80 | 2.4.5                 | com.fasterxml.jackson.core.jackson-annotations
169 | Active   |  80 | 2.4.5                 | com.fasterxml.jackson.core.jackson-core
170 | Active   |  80 | 2.4.5                 | com.fasterxml.jackson.core.jackson-databind
171 | Active   |  80 | 2.4.5                 | com.fasterxml.jackson.dataformat.jackson-dataformat-yaml
172 | Active   |  80 | 1.5.8                 | io.swagger.core
173 | Active   |  80 | 1.5.8                 | io.swagger.jaxrs
174 | Active   |  80 | 1.5.8                 | io.swagger.models
175 | Active   |  80 | 3.5.0                 | javax.jmdns
176 | Active   |  80 | 3.4.0                 | org.apache.commons.lang3
178 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.ui.basic, Fragments: 182
179 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.ui.classic
180 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.ui.paper, Fragments: 185
181 | Active   |  80 | 2.0.0.201607210102    | org.openhab.io.rest.docs
182 | Resolved |  80 | 2.0.0.201607210102    | org.openhab.ui.basicui, Hosts: 178
183 | Active   |  80 | 2.0.0.201607210102    | org.openhab.ui.classicui
184 | Active   |  80 | 2.0.0.201607210102    | org.openhab.ui.dashboard
185 | Resolved |  80 | 2.0.0.201607210102    | org.openhab.ui.paperui, Hosts: 180
186 | Active   |  80 | 0.9.10.v20160429-1435 | reflections
187 | Active   |  80 | 1.5.8.v20160511-1038  | swagger-jersey2-jaxrs
188 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.binding.ntp
189 | Active   |  80 | 2.0.0.201607290102    | org.openhab.binding.astro
190 | Active   |  80 | 1.9.0.201607280110    | org.openhab.binding.netatmo
191 | Active   |  80 | 2.0.0.201607210102    | org.openhab.core.compat1x
192 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.transform.map
193 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.transform.regex
194 | Active   |  80 | 1.9.0.201607280110    | org.openhab.binding.mios
195 | Active   |  80 | 1.9.0.201607280110    | org.openhab.binding.mqtt
196 | Active   |  80 | 1.9.0.201607280110    | org.openhab.io.transport.mqtt
197 | Active   |  80 | 1.9.0.201607280110    | org.openhab.binding.weather
198 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.binding.hue
199 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.io.transport.upnp
200 | Active   |  80 | 2.0.0.201607290102    | org.openhab.binding.rfxcom
201 | Active   |  80 | 2.0.0.201607210102    | org.openhab.io.transport.serial
202 | Active   |  80 | 1.9.0.201607280110    | org.openhab.binding.powermax
203 | Active   |  80 | 2.0.0.201607290102    | org.openhab.binding.freebox
204 | Active   |  80 | 1.9.0.201607280110    | org.openhab.persistence.rrd4j
205 | Active   |  80 | 1.9.0.201607280110    | org.openhab.action.mios
206 | Active   |  80 | 0.9.0.201607271614    | org.eclipse.smarthome.transform.scale
207 | Active   |  80 | 0.9.0.201607292000    | org.eclipse.smarthome.binding.sonos
208 | Active   |  80 | 0.9.0.201607292343    | org.eclipse.smarthome.io.transport.mdns
```
and log warnings:
```
2016-07-30 03:36:24.447 [WARN ] [home.core.internal.items.ItemUpdater] - InstantiationException on org.eclipse.smarthome.core.library.types.StringType
2016-07-30 03:36:24.490 [WARN ] [home.core.internal.items.ItemUpdater] - InstantiationException on org.eclipse.smarthome.core.library.types.StringType
```

**Comment by kaikreuzer** [comment]
The last stable ESH build is from July 27, while this fix only went into the nightly of July 28. So you will have to wait for the next [stable build](https://hudson.eclipse.org/smarthome/job/SmartHomeDistribution-Stable/changes) in order to find this fix in openHAB.
